### DISCUSSION TITLE
🆕 Log in with multiple GitHub accounts on GitHub Mobile 📱

### DISCUSSION THREAD

**Discussion by Akash1134** [community/community#63893] [Mobile]
<img width="1365" alt="Screenshot 2023-08-16 at 11 36 48 AM" src="https://github.com/community/community/assets/95834130/ef104b3a-6a75-4aba-bf40-a2667ddf9900">

Introducing support for multiple GitHub accounts within GitHub Mobile! Log in with your work and personal accounts to stay in touch with your projects, wherever they're happening. 🏁 

- To add multiple accounts to GitHub Mobile, either navigate to **Profile > Settings > Accounts**, or **long-press on the Profile tab** to get to the account switcher. See the number of unread notifications across each account, swap to another account, or sign in or out of accounts. 

- **Receive push notifications** for each account, with just the right amount of context to keep you focused on the work that matters. Keep your data separate between each account, ensuring the right accounts are active when viewing private content.

**Download or update GitHub Mobile today** from the [Apple App Store](https://apps.apple.com/us/app/github/id1477376905) or [Google Play Store](https://play.google.com/store/apps/details?id=com.github.android) to get started. 🚀

**Comment by Akash1134** [comment]
🚀

**Reply by Diego200208** [reply]
https://linkjust.com/Alexis1997

**Comment by bronglil** [comment]
🚀

**Comment by Brandon170699Vc** [comment]
> [bdalalhaljwysm@gmail.com](mailto:bdalalhaljwysm@gmail.com)

**Comment by Mohammed71o** [comment]
> 🚀

**Comment by Mohammed71o** [comment]
> 🚀

**Comment by Mohammed71o** [comment]
> > [bdalalhaljwysm@gmail.com](mailto:bdalalhaljwysm@gmail.com)
> 
>

**Comment by Mohammed71o** [comment]
> > [bdalalhaljwysm@gmail.com](mailto:bdalalhaljwysm@gmail.com)
> 
>

**Comment by Mohammed71o** [comment]
> > [bdalalhaljwysm@gmail.com](mailto:bdalalhaljwysm@gmail.com)
> 
>

**Comment by Mohammed71o** [comment]
Ses quoi sa cette liste là des dossiers d'enfants ou quoi

Le mar. 10 oct. 2023, 00 h 50, Rishi Mondal ***@***.***> a
écrit :

**Comment by Mohammed71o** [comment]
Ses dans l'ordi,sa bien j' ai payé cette ordi,mes venez la chercher,
j'espère que quelqu'un va me donner quelques choses.
Mais bon êtes vous proche ou quoi..

**Comment by Mohammed71o** [comment]
Oui non , vous êtes proche ou non

**Comment by kauaneiras** [comment]
😮

**Comment by kbqdev** [comment]
🚀🚀🚀
